src/SFA.DAS.LevyTransferMatching.Infrastructure/Dto/AccountDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.LevyTransferMatching.Infrastructure.Dto
+{
+    public class AccountDto
+    {
+        public double RemainingTransferAllowance { get; set; }
```

Comment:
Is this a monetary value, should it be decimal rather than double? Do we do monetary calculations on it or is it just an indicative value?

Reply:
I think it's defined as a double in the source (MA API)